omod-2.2/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_2/FulfillerDetailsRepresentation.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs2_2;
+
+import org.openmrs.Order;
+
+public class FulfillerDetailsRepresentation {
+	
+	private Order.FulfillerStatus fulfillerStatus;
+	private String fulfillerComment;
+	private Order parent;
```

Comment:
Can you please rename this to order for clarity?

Reply:
I am not sure what you mean? Should the order not be called parent?

Reply:
The field value is an Order object, therefore it's more intuitive to name it order